controller/iteration.go

```diff
@@ -215,7 +215,7 @@ func ConvertIteration(request *goa.RequestData, itr *iteration.Iteration, additi
 		Relationships: &app.IterationRelations{
 			Space: &app.RelationGeneric{
```

Comment:
Because we have two type of objects for the relation, relationSpace and relationGeneric. I could unify this in a different PR.